examples/client-transports.rs

```diff
@@ -337,6 +337,8 @@ where
         + Sync
         + 'static,
 {
+    use domain::net::xfr::processing::XfrResponseProcessor;
```

Comment:
Maybe this should go into a separate example and limit client-transports to stuff in net::client?

Reply:
Okay. Maybe that has value, not sure as it would still need to use a client as it is about processing a client response. In this case I thought it helped to make the tsig axfr example output more useful.

I could instead make the client-transport tsig example dump out zonefile text format rrs instead of what it currently prints, and leave the use of xfr response processing to the zone maintainer which actually processes the result of transfers and applies them.

Reply:
Yes, that sounds good

Reply:
I'll just simplify this to be consistent with the rest of this example so that it just debug prints the replies and nothing more.